saleor/graphql/checkout/mutations/order_create_from_checkout.py

```diff
@@ -62,6 +62,7 @@ class Meta:
         object_type = Order
         permissions = (CheckoutPermissions.HANDLE_CHECKOUTS,)
         error_type_class = OrderCreateFromCheckoutError
+        metadata_permissions_map_key = "Checkout"
```

Comment:
Do you need this? Can't we determine a type name in different way? Like using `id` field that is `global` and contain a `typename`?